app/controlplane/internal/service/auth.go

```diff
@@ -242,6 +249,11 @@ func callbackHandler(svc *AuthService, w http.ResponseWriter, r *http.Request) (
 		svc.log.Infow("msg", "new user associated to an org", "org_id", currentOrg.ID, "user_id", u.ID)
 	}
 
+	// Auto onboard the user to the organizations specified in the config
+	if err := autoOnboardOnOrganizations(ctx, svc, u); err != nil {
```

Comment:
nitpick. `AutoOnboardOrganizations` sound better to me.

Reply:
this whole method could probably be part of a biz user-case instead no? The less logic we have in svc the better, and this module is already pretty overloaded. 

so yes, I'd move this method to biz under organization biz use-cases

the signature could be smth like 
```
func (s *..usecase) autoOnboardOnOrganizations(ctx context.Context, orgNames []string, userID string) error
```

note how I am removing depending on objects and instead make sure we pass just the attributes, i.e userID instead of the whole user, this will help with testing later on.